README.md

```diff
@@ -105,6 +105,14 @@ If you happen to use [`fd`](https://github.com/sharkdp/fd), you can use the `-X`
 fd … -X bat
 ```
 
+#### `ripgrep`
+
+With [`batgrep`](https://github.com/eth-p/bat-extras/blob/master/doc/batgrep.md), `bat` can be used as the printer for [`ripgrep`](https://github.com/BurntSushi/ripgrep) search results.
+
+```bash
+batgrep Printer src/
```

Comment:
This might be slightly confusing, as you mention "printer" above. Maybe `batgrep needle src/` or `batgrep pattern [<path>]`?

Reply:
Hmm... On a second look, that is confusing. The latter would probably be the better example